policyengine_us/variables/gov/states/mt/tax/income/mt_applicable_ald_deductions.py

```diff
@@ -0,0 +1,19 @@
+from policyengine_us.model_api import *
+
+
+class mt_applicable_ald_deductions(Variable):
+    value_type = float
+    entity = Person
+    label = "Montana applicable above-the-line deductions "
+    unit = USD
+    definition_period = YEAR
+
+    def formula(person, period, parameters):
+        ald_deductions = person.tax_unit("above_the_line_deductions", period)
+        spouse = person("is_tax_unit_spouse", period)
```

Comment:
```suggestion
        total_deduction = ald_deductions / person.tax_unit("num", period)
```

Reply:
I think dependents should not have ald_deductions. Also, I can't find the file for "num", is it the same as the "tax_unit_size"?

Reply:
Sorry, it's now head_spouse_count

Reply:
I would keep the condition for "is_tax_unit_head_or_spouse" to avoid assigning values to dependents